consensus/expected.go

```diff
@@ -247,19 +247,6 @@ func (c *Expected) RunStateTransition(ctx context.Context, ts types.TipSet, ance
 		return nil, err
 	}
 
-	sl := ts.ToSlice()
```

Comment:
breadcrumb: This appears to be removed because the TipSet constructor doesn't permit inconsistent blocks. Note there is a small inconsistency in that this compares duplicates blocks by block.String (cid + json) whereas NewTipSet only compares by cid. Comparing by CID seems better.